packages/rum-core/src/domain/action/actionCollection.spec.ts

```diff
@@ -1,29 +1,45 @@
-import { type Duration, type RelativeTime, type ServerDuration, type TimeStamp } from '@datadog/browser-core'
-import { createNewEvent } from '@datadog/browser-core/test'
-import type { RawRumActionEvent } from '@datadog/browser-rum-core'
-import type { TestSetupBuilder } from '../../../test'
-import { setup } from '../../../test'
+import type { Duration, RelativeTime, ServerDuration, TimeStamp } from '@datadog/browser-core'
+import { noop, Observable } from '@datadog/browser-core'
+import { createNewEvent, registerCleanupTask } from '@datadog/browser-core/test'
+import type { RawRumActionEvent, RawRumEventCollectedData, RumConfiguration } from '@datadog/browser-rum-core'
+import { validateRumEventFormat } from '../../../test'
+import type { RawRumEvent } from '../../rawRumEvent.types'
 import { RumEventType, ActionType } from '../../rawRumEvent.types'
-import { LifeCycleEventType } from '../lifeCycle'
+import { LifeCycle, LifeCycleEventType } from '../lifeCycle'
+import type { PageStateHistory } from '../contexts/pageStateHistory'
 import { startActionCollection } from './actionCollection'
 
+const pageStateHistory: PageStateHistory = {
```

Comment:
**💬 suggestion:** ‏it looks like PageStateHistory is mocked 6 times in tests. Could we refactor it? (ex: `createMockPageStateHistory(options)` in packages/rum-core/test/mockPageStateHistory.ts)